lms/envs/devstack.py

```diff
@@ -141,3 +141,7 @@ def should_show_debug_toolbar(_):
 MODULESTORE = convert_module_store_setting_if_needed(MODULESTORE)
 
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+FEATURES['ENABLE_COMBINED_LOGIN_REGISTRATION'] = True
+FEATURES['AUTOMATIC_AUTH_FOR_TESTING'] = True
+FEATURES['ENABLE_DISCUSSION_SERVICE'] = False
```

Comment:
Is this a change we want to make for everyone?  Enabling combined login/registration might make it difficult for open source contributors to test theming on the open source version of the login and registration forms.

If we do want to make these permanent, we should move these lines about line 133 so we don't wipe out local developer overrides.